integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/api/CrowdNodeApi.kt

```diff
@@ -258,6 +259,7 @@ class CrowdNodeApiAggregator @Inject constructor(
                 } catch (ex: Exception) {
                     handleError(ex, appContext.getString(R.string.crowdnode_withdraw_error))
                 }
+                refreshBalance(retries = 3)
```

Comment:
This doesn't catch the withdrawal transaction that has the amount specified by the user.

Reply:
Right, we can track `CrowdNodeWithdrawalReceivedTx` here as well for a more robust solution. See DMs